### DISCUSSION TITLE
Definition of an addict

### DISCUSSION THREAD

**Discussion by msaivepinkall** [flux511/Personality-vs-DrugUse#11] [Ideas]
Which drugs not to consider (because socially accepted):
- caff
- cannabis
- alcohol
- choc

Definition of a drug addict:
- CL5: consumed during the last week
- CL6: consumed during the last day

**Comment by flux511** [comment]
the drug "Semer" from the dataframe should be removed:
Ref. "Nine of these persons were found to also have endorsed using a fictitious recreational
drug, and which was included precisely to identify respondents who over-claim, as have other studies of this kind"
